Britannia/localization/english/event_localization/BTWK_events_l_english.yml

```diff
@@ -1,25 +1,50 @@
+  BTWK_frontier_event.0201.a:0 "Out with the old..."
+  BTWK_frontier_event.0201.a_tt:0 "All instances of the [special_buildings|E] #italic Hadrian's Wall#! or #italic Hadrian's Wall - Restored#! will be ruined."
+
+  BTWK_frontier_event.0202:0 "The End of Hadrian's Wall"  # Every other player
+  BTWK_frontier_event.0202_desc_header:0 "[GetScope(destroyed_hadrians_wall_char).Char.GetName] has ordered the demolition of Hadrian's Wall"
+  BTWK_frontier_event.0202_desc_header_vassal:0 "Our liege, [GetScope(destroyed_hadrians_wall_char).Char.GetName], has ordered the demolition of Hadrian's Wall."
+  BTWK_frontier_event.0202_desc_main:0 "\n\n [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] hopes that now the wall no longer serves any meaningful purpose, its stone could be repurposed for the developing settlements throughout [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] realm."
+  BTWK_frontier_event.0202_desc_imperial_like_destroyer:0 "[GetScope(destroyed_hadrians_wall_char).Char.GetFirstName] has done this despite [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] his heritage, from which is intrinsically linked to the momument. For this act, many of his kinsmen have been severely angered."
+  BTWK_frontier_event.0202_desc_else_destroyer:0 "Having no ancient ties to the land, and the wall which sits atop it, [GetScope(destroyed_hadrians_wall_char).Char.GetFirstNamePossessiveNoTooltipRegnal] decision was  tenable. The act has angered the likes of the Britons and of the Latins, who have ancient ties to the monument."
+  BTWK_frontier_event.0202.a_imperial_like:0 "A terrible day for Britannia"
+  BTWK_frontier_event.0202.a_other_0 "Interesting news" 
+  BTWK_frontier_event.0202.a_other_vassal:0 "For the good of the realm!"
+
+  BTWK_frontier_event.0203:0 "Hadrian's Wall Restored"  # ROOT of restoration
+  BTWK_frontier_event.0203_desc:0 "The conclusion of the renovations in [GetGlobalVar('last_old_hadrians_wall_province').MakeScope.Province.GetProvince] has marked the complete restoration of Hadrian's Wall. While the gesture was ultimately symbolic as a reminder of the Imperial roots all Britons share, the wall is now operable as the defensive frontline as it once was, should the need ever arise again. \n\nThough the effort to rebuild has taken many years, gold and manpower to complete, as the overseer of the final stones being set into place, you are the one the poems shall remember the most. Many among the isle are beginning to praise you as 'Hadrianus', as a successor to the great Emperor Hadrian himself. Perhaps this new reputation might aid your bids to become High King of Britannia, should you wish it..."
+  BTWK_frontier_event.0203.a:0 "OPTION"  # Not sure on this
```

Comment:
A generic "For Britannia" or something similar in Welsh?

Reply:
I'm seeing an "Ar gyfer Prydain!". Is this what you mean? May not be applicable if the character is not Brythonic, so maybe have a Latin option as well?

Reply:
I might ask the Discord see if there are any Welsh people that can translate for me lol

Reply:
Oh I wasn't thinking of anything in particular, just spitballing since you weren't sure what to do. Generally for this kind of thing I either just do something generic or try to do some wordplay (or even not wordplay if it fits) on relevant mottos or song/poem lines.

Reply:
Could do something with "Cymru am byth" (Prydain am byth?) though it appears to be a bit modern, or possibly something with the Red Dragon... unless you want to save them for some bigger events like uniting Britannia or something as they're a bit hard hitting

Reply:
I feel like something like that would be good for something "bigger" as you say. This is really just a symbolic thing, as we've established. I will use it for now as a placeholder & mark it as such though, given I haven't got anything better.